File: MODUL 3/PRAK304-2410817210005-Muhammad_Irgi_Fahrezha.py
bilangan = int(input())

if bilangan >= 1 and bilangan <= 9:
    print("Satuan")

elif bilangan >= 11 and bilangan <= 19:
    print("Belasan")

elif bilangan >= 20 and bilangan <= 99 or bilangan == 10:
    print("Puluhan")

elif bilangan == 0:
    print("Nol")

else: 
    print("Anda Menginput Melebihi Limit Bilangan")

File: MODUL 3/PRAK304-2410817210005-Muhammad_Irgi_Fahrezha.c
#include <stdio.h>

int main () {

    int bilangan;

    scanf("%d", &bilangan);

    if (bilangan >= 1 && bilangan <= 9) {
        printf("Satuan");
    }

    else if (bilangan >= 11 && bilangan <=19) {
        printf("Belasan");
    }

    else if (bilangan >= 20 && bilangan <=99 || bilangan == 10) {
        printf("Puluhan");
    }

    else if (bilangan == 0) {
        printf("Nol");
    }

    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}